### PR TITLE
Adjust deprovision playbook

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: playbooks/deprovision.yaml [hunk 1]
```diff
@@ -12,81 +12,7 @@
...
-      - "buildconfigs/result-api"
-      - "svc/result-api"
-      - "svc/janusgraph"
-      - "deploymentconfigs/result-api"
-      - "deploymentconfigs/janusgraph"
-    ignore_errors: true
+  - name: Deprovision Thoth
+    # We delete all resources/components except PVC that do not have app=thoth
+    # labels assigned to make sure we do not lose data.
+    command: oc delete --selector 'app=thoth' --all
```
Comment (goern) [inline playbooks/deprovision.yaml]: I think janusgraph is not hit with that label?!

Comment (fridex) [reply playbooks/deprovision.yaml]: No, its not. Should I add it as a new task?

**Comment by fridex** [comment]
Ready for a review - tested bringing the app up, down and up again. I've removed PV for janusgraph for now as if enabled ES fails to start.

File: playbooks/deprovision.yaml [hunk 2]
```diff
@@ -12,81 +12,11 @@
...
-      - "package-extract-buildconfig"
-      - "package-releases-buildconfig"
-      - "package-releases-cronjob"
-      - "cleanup-imagestream"
-      - "graph-refresh-imagestream"
-      - "graph-sync-imagestream"
-      - "package-releases-imagestream"
-    ignore_errors: true
+  - name: Deprovision Thoth
+    command: "oc delete bc,configmap,cronjob,dc,is,pod,role,rolebinding,route,\
```
Comment (goern) [inline playbooks/deprovision.yaml]: if you deleted the dc all the pods should be gone...

File: playbooks/deprovision.yaml [hunk 3]
```diff
@@ -12,81 +12,11 @@
...
+      secret,svc,sa,template,clusterrole --selector 'app=thoth'"
```
Comment (goern) [inline playbooks/deprovision.yaml]: are we always allowed to delete the clusterrole?!

File: playbooks/deprovision.yaml (hunk 2 above)
Comment (fridex) [reply playbooks/deprovision.yaml]: > if you deleted the dc all the pods should be gone...

But only for objects that have dc defined. When we run solver, adviser, package extract, these are run dynamically from user-api and do not have dc defined => these pods will not be deleted.

File: playbooks/deprovision.yaml (hunk 3 above)
Comment (fridex) [reply playbooks/deprovision.yaml]: If not, there will be an error message printed and others will (?) be deleted as expected.

Comment (fridex) [reply playbooks/deprovision.yaml]: BTW, this is command from which I got the list (it was not manually curated, I just adjusted some names):

```console
find -iname '*.yaml' -exec grep 'kind:' {} \; | cut -d: -f2 | sort -u | tr '[:upper:]' '[:lower:]'  | tr '\n' ',' | tr -d '[:space:]'
```

Comment (goern) [reply playbooks/deprovision.yaml]: !

**Pull request merged**